Data Structures/array implementation of stack.c

```diff
@@ -0,0 +1,80 @@
+#include<stdio.h>
+#include<conio.h>
+#define MAX50
+int stack [MAX + 1], top=0;
+
+// MAIN Function
+
+void main()
+{
+  clrscr();
+  void create(), traverse(), push(), pop();
+      
+  create();                         // Calling Create function
+  printf("\n stack is: \n");
+  traverse();
+  
+  push();                                 // Calling PUSH function to enter elements
+  printf("After Push the element in the stack is: \n");
+  traverse();
+  
+  pop();                                    // Calling POP function to delete elements
+  printf("After pop the element in the stack is: \n");
+  traverse();                                           // Calling traverse function
+  
+  getch();
+}
+
+// Function to create stack
+
+void create()
+{
+   char ch;
+  do
+  {
+    top++;
+    printf("Input Element");
+    scanf("%d", stack[top]);
```

Comment:
This will throw an error when the top becomes equal to the size of the array.  We should add check like this:
```
if(top==MAX)
     printf("Stack is full");
else
   scanf("%d", &stack[top]);
```